src/Directory.Build.props

```diff
@@ -48,7 +48,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <LangVersion>10.0</LangVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
Is 11.0 not supported ? (I'd rather have it be explicit so we don't fall back on what the current targetframework supports).

Reply:
@jeromelaban It's supported but not for all projects it seems.

Reply:
I'll see if I could use 11 here and adjust the failing projects.